Population.cpp

```diff
@@ -23,3 +26,111 @@ const Individual &Population::operator[](int index) const {
     return population_vector_[index];
 }
 
+Individual Population::generate_individual() {
```

Comment:
generate_individual() is not a good name for this method since it selects an individual, rather than generating it.